scraper.go

```diff
@@ -78,6 +78,15 @@ var (
 		},
 		[]string{"id", "type"},
 	)
+	canBindGauge = prometheus.NewGaugeVec(
```

Comment:
This should really be a counter. You are storing successful and failed bind attempts. These can be counts rather than gauge values.